src/components/Layout/Foot.vue

```diff
@@ -1,10 +1,6 @@
 <template>
   <footer class="footer">
   <v-footer color="primary lighten-1">
-    <v-container>
```

Comment:
I think the container is still needed for giving the footer content a width, it's just `col` and `row` that are superfluous.

![image](https://github.com/wbstack/ui/assets/1662740/55efa0c4-5d3a-4e88-9e88-f28b1d3251ea)